podcastfy/core/character.py

```diff
@@ -0,0 +1,30 @@
+from typing import Dict, Optional
+
+from podcastfy.core.tts_configs import TTSConfig
+
+
+class Character:
+    """Represents a character in the podcast."""
+
+    def __init__(self, name: str, role: str, tts_configs: Dict[str, TTSConfig] = {},
+                 default_description_for_llm: str = ""):
+        self.name = name
+        self.role = role
+        self.tts_configs = tts_configs
+        self.default_description_for_llm = default_description_for_llm
+        self.preferred_tts = next(iter(tts_configs.keys()), None)  # Set first TTS as default, can be None
```

Comment:
why should a character be aware of TTS models?
A Character should describe attributes and behavior of a Podcast participant.
I'd argue perhaps only the Transcript generation should be aware of Character information.